lib/internal/policy/manifest.js

```diff
@@ -500,13 +512,25 @@ class Manifest {
     ObjectFreeze(this);
   }
 
+  /**
+   * @param {string} href
+   * @returns {string}
+   */
+  getBaseURL(href) {
+    if (this.#resourceBaseURLs.has(href)) {
+      return this.#resourceBaseURLs.get(href);
+    }
+    return href;
```

Comment:
would there ever be a nullish base URL? if not, why not:
```suggestion
    return this.#resourceBaseURLs.get(href) ?? href;
```

Reply:
we should remove this to its own PR, got mixed in while I was doing `git` jargon. It can be `undefined` in this version though the idea of it being `null` is a bit we probably should reserve in w/e the future PR is.